keyboards/handwired/yamsek/info.json

```diff
@@ -0,0 +1,77 @@
+                {"matrix":[4,4], "label":"R", "x":4, "y":0.125},
+                {"matrix":[4,5], "label":"T", "x":5, "y":0.25},
+                {"matrix":[5,0], "label":"Tab", "x":0, "y":1.75},
+                {"matrix":[5,1], "label":"A", "x":1, "y":1.375},
+                {"matrix":[5,2], "label":"S", "x":2, "y":1.125},
+                {"matrix":[5,3], "label":"D", "x":3, "y":1},
+                {"matrix":[5,4], "label":"F", "x":4, "y":1.125},
+                {"matrix":[5,5], "label":"G", "x":5, "y":1.25},
+                {"matrix":[6,0], "label":"LShift", "x":0, "y":2.75},
+                {"matrix":[6,1], "label":"Z", "x":1, "y":2.375},
+                {"matrix":[6,2], "label":"X", "x":2, "y":2.125},
+                {"matrix":[6,3], "label":"C", "x":3, "y":2},
+                {"matrix":[6,4], "label":"V", "x":4, "y":2.125},
+                {"matrix":[6,5], "label":"B", "x":5, "y":2.25},
+                {"matrix":[7,0], "label":"TG(Game)", "x":5.5, "y":5},
+                {"matrix":[7,1], "label":"Mo(NPad)", "x":4.5, "y":4.6},
+                {"matrix":[7,2], "label":"Backs", "x":4.5, "y":4},
+                {"matrix":[7,3], "label":"Alt", "x":5.7, "y":3.5},
+                {"matrix":[7,4], "label":"Ctrl", "x":6.75, "y":3.75},
+                {"matrix":[7,5], "label":"Shift", "x":7.75, "y":4.2},
+                {"matrix":[0,0], "label":"Y", "x":13, "y":0.25},
+                {"matrix":[0,1], "label":"U", "x":14, "y":0.125},
+                {"matrix":[0,2], "label":"I", "x":15, "y":0},
+                {"matrix":[0,3], "label":"O", "x":16, "y":0.125},
+                {"matrix":[0,4], "label":"P", "x":17, "y":0.375},
+                {"matrix":[0,5], "label":"-", "x":18, "y":0.75},
+                {"matrix":[1,0], "label":"H", "x":13, "y":1.25},
+                {"matrix":[1,1], "label":"J", "x":14, "y":1.125},
+                {"matrix":[1,2], "label":"K", "x":15, "y":1},
+                {"matrix":[1,3], "label":"L", "x":16, "y":1.125},
+                {"matrix":[1,4], "label":":", "x":17, "y":1.375},
+                {"matrix":[1,5], "label":"'", "x":18, "y":1.75},
+                {"matrix":[2,0], "label":"N", "x":13, "y":2.25},
+                {"matrix":[2,1], "label":"M", "x":14, "y":2.125},
+                {"matrix":[2,2], "label":"<", "x":15, "y":2},
+                {"matrix":[2,3], "label":">", "x":16, "y":2.125},
+                {"matrix":[2,4], "label":"?", "x":17, "y":2.375},
+                {"matrix":[2,5], "label":"=", "x":18, "y":2.75},
+                {"matrix":[3,0], "label":"Shift", "x":10, "y":4.2},
+                {"matrix":[3,1], "label":"Enter", "x":11, "y":3.75},
+                {"matrix":[3,2], "label":"Spac", "x":11.95, "y":3.5},
+                {"matrix":[3,3], "label":"Del", "x":13.2, "y":4},
+                {"matrix":[3,4], "label":"Mo(Arr)", "x":12.2, "y":5},
+                {"matrix":[3,5], "label":"Mo(Fun)", "x":13.2, "y":4.6},
```

Comment:
OK, I tried to fix it to match the keymap (assuming that the labels here were correct):
```suggestion
                {"matrix":[4,0], "label":"Esc", "x":0, "y":0.75},
                {"matrix":[4,1], "label":"Q", "x":1, "y":0.375},
                {"matrix":[4,2], "label":"W", "x":2, "y":0.125},
                {"matrix":[4,3], "label":"E", "x":3, "y":0},
                {"matrix":[4,4], "label":"R", "x":4, "y":0.125},
                {"matrix":[4,5], "label":"T", "x":5, "y":0.25},
                {"matrix":[0,0], "label":"Y", "x":13, "y":0.25},
                {"matrix":[0,1], "label":"U", "x":14, "y":0.125},
                {"matrix":[0,2], "label":"I", "x":15, "y":0},
                {"matrix":[0,3], "label":"O", "x":16, "y":0.125},
                {"matrix":[0,4], "label":"P", "x":17, "y":0.375},
                {"matrix":[0,5], "label":"-", "x":18, "y":0.75},
                {"matrix":[5,0], "label":"Tab", "x":0, "y":1.75},
                {"matrix":[5,1], "label":"A", "x":1, "y":1.375},
                {"matrix":[5,2], "label":"S", "x":2, "y":1.125},
                {"matrix":[5,3], "label":"D", "x":3, "y":1},
                {"matrix":[5,4], "label":"F", "x":4, "y":1.125},
                {"matrix":[5,5], "label":"G", "x":5, "y":1.25},
                {"matrix":[1,0], "label":"H", "x":13, "y":1.25},
                {"matrix":[1,1], "label":"J", "x":14, "y":1.125},
                {"matrix":[1,2], "label":"K", "x":15, "y":1},
                {"matrix":[1,3], "label":"L", "x":16, "y":1.125},
                {"matrix":[1,4], "label":":", "x":17, "y":1.375},
                {"matrix":[1,5], "label":"'", "x":18, "y":1.75},
                {"matrix":[6,0], "label":"LShift", "x":0, "y":2.75},
                {"matrix":[6,1], "label":"Z", "x":1, "y":2.375},
                {"matrix":[6,2], "label":"X", "x":2, "y":2.125},
                {"matrix":[6,3], "label":"C", "x":3, "y":2},
                {"matrix":[6,4], "label":"V", "x":4, "y":2.125},
                {"matrix":[6,5], "label":"B", "x":5, "y":2.25},
                {"matrix":[2,0], "label":"N", "x":13, "y":2.25},
                {"matrix":[2,1], "label":"M", "x":14, "y":2.125},
                {"matrix":[2,2], "label":"<", "x":15, "y":2},
                {"matrix":[2,3], "label":">", "x":16, "y":2.125},
                {"matrix":[2,4], "label":"?", "x":17, "y":2.375},
                {"matrix":[2,5], "label":"=", "x":18, "y":2.75},
                {"matrix":[7,0], "label":"TG(Game)", "x":5.5, "y":5},
                {"matrix":[7,1], "label":"Mo(NPad)", "x":4.5, "y":4.6},
                {"matrix":[7,2], "label":"Backs", "x":4.5, "y":4},
                {"matrix":[7,3], "label":"Alt", "x":5.7, "y":3.5},
                {"matrix":[7,4], "label":"Ctrl", "x":6.75, "y":3.75},
                {"matrix":[7,5], "label":"Shift", "x":7.75, "y":4.2},
                {"matrix":[3,0], "label":"Shift", "x":10, "y":4.2},
                {"matrix":[3,1], "label":"Enter", "x":11, "y":3.75},
                {"matrix":[3,2], "label":"Spac", "x":11.95, "y":3.5},
                {"matrix":[3,3], "label":"Del", "x":13.2, "y":4},
                {"matrix":[3,4], "label":"Mo(Arr)", "x":12.2, "y":5},
                {"matrix":[3,5], "label":"Mo(Fun)", "x":13.2, "y":4.6},
```
(Having a trailing comma is so much nicer to work with; pure JSON does not allow that, but `info.json` files are actually read as Hjson, which supports that feature.)

The bottom row ordering is still somewhat weird though (it does not match the physical layout); but fixing that would require changing the keymap to match.  Check the ordering and coordinates of the last two keys though (`[3,4]` and `[3,5]`) — the diagram in `keymap.c` shows that these keys are ordered right to left, but the coordinates here show the left to right ordering.

But maybe it would make sense to align the order of keys with their physical locations and change from this:
![image](https://user-images.githubusercontent.com/616082/233855022-1a16b552-64b5-4cc7-a510-d7f3c9db8bf9.png)
to this:
![image](https://user-images.githubusercontent.com/616082/233855043-50df252b-7491-4492-9225-bbc44ed1ab3a.png)
(aligning the keys labeled 9…12 as a separate row).

Reply:
BTW, if you actually try to compile the firmware with these changes, any mismatch between `info.json`, the real wiring and the actual usage of the `LAYOUT()` macro in the keymap should show up as some keys getting swapped around or not working at all.

Reply:
doesn't the ordering have to be aligned with the actual wiring?
the way i handwired mine was as 6colums x 4rows per halve. with the thumbcluster keys being "uncurled" clockwise into one single row. easy to visualize/learn/no gaps in the in-memory-matrix/...

Reply:
`matrix: [row, col]` reflects the wiring; entry positions in the layout array correspond to the argument positions in the layout macro.  The usual QMK convention is that the order of arguments in the layout macro is chosen to resemble the physical keyboard layout (with some limitations imposed by the text representation — e.g., for a column-staggered keyboard like this the rows would be curved on the real keyboard, but straight in the text representation).

Actually your old commits before the rebase had this layout macro:
https://github.com/qmk/qmk_firmware/blob/9fb72a10c98af31e3e059831e5b07636c44b484c/keyboards/handwired/yamsek/yamsek.h#L18-L36
which looks very similar to the my second suggestion above (note that the suggested edit does not include that change for the thumb cluster, because it would also require editing the keymap accordingly).